pset-functions.js

```diff
@@ -76,6 +90,55 @@ console.log('Problem 2:')
 
 // Add your code below this line
 
+// Letter                           Value
+// A, E, I, O, U, L, N, R, S, T       1
+// D, G                               2
+// B, C, M, P                         3
+// F, H, V, W, Y                      4
+// K                                  5
+// J, X                               8
+// Q, Z                               10
+
+function scrabbleScore(word){
+
+  const wordArray = word.toUpperCase().split("")
+  let sum = 0
+
+  const value1 = ["A", "E", "I", "O", "U", "L", "N", "R", "S", "T"]
+  const value2 = ["D", "G"]
+  const value3 = ["B", "C", "M", "P"]
+  const value4 = ["F", "H", "V", "W", "Y"]
+  const value5 = ["K"]
+  const value8 = ["J", "X"]
+  const value10 = ["Q", "Z"]
+
+wordArray.forEach((current) => {
+  //no return
+  if (["A", "E", "I", "O", "U", "L", "N", "R", "S", "T"].includes(current)) {
+    sum+=1
```

Comment:
Way to keep edge cases in mind on line 104! Also, great combo of the forEach iterator with the .includes method for checking scores.